test/library/packages/LinearAlgebra/performance/lu/lu-perf.chpl

```diff
@@ -0,0 +1,51 @@
+use Time;
+use Math;
+use LinearAlgebra;
+use LinearAlgebra.Sparse;
+
+config const m=1000,
+             iters=10,
+             /* Omit timing output */
+             correctness=false;
+             
+config type eltType = real;
+
+const nbytes = numBytes(eltType);
+
+proc main() {
+
+  const Space = {1..m, 1..m};
+
+  var t: Timer;
+
+  if !correctness {
+    writeln('=========================================');
+    writeln('  LU Factorization Performance Test');
+    writeln('=========================================');
+    writeln('iters : ', iters);
+    writeln('m     : ', m);
+    writeln('numLocales     : ', numLocales);
+    writeln('MB    : ', (nbytes*m*m) / 10**6);
+    writeln();
+  }
+
+  var A: [Space] eltType;
+
+  addDiag(A, 0, 2);
+  addDiag(A, 1, -1);
+  addDiag(A, -1, -1);
+  addDiag(A, 10, 0.5);
+  addDiag(A, -10, 0.5);
```

Comment:
It is OK that this PR is dependent on your PR where you add these functions. But let's update these to setDiag as it is the interface that we settled on.